regression/goto-harness/chain.sh

```diff
@@ -8,13 +8,20 @@ cbmc=$3
 is_windows=$4
 entry_point='generated_entry_function'
 
+shift 4
 name=${*:$#}
 name=${name%.c}
-args=${*:5:$#-5}
 
 input_c_file="${name}.c"
 input_goto_binary="${name}.gb"
-harness_c_file="${name}-harness.c"
+if [[ "$1" == "harness.gb" ]]; then
```

Comment:
One of these days we should really decide on the preferred file ending for goto-model binaries, `.gb`? `.goto`? Something else?

Reply:
Done: #5826

Reply:
Thanks.